app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabs.kt

```diff
@@ -186,53 +184,6 @@ fun SyncedTabsSectionHeader(
     }
 }
 
-/**
- * Synced tab list item UI
- *
- * @param tabTitleText The tab's display text.
- * @param url The tab's URL.
- * @param onClick The click handler when this synced tab is clicked.
- */
-@Composable
-fun SyncedTabsTabItem(
-    tabTitleText: String,
-    url: String,
-    onClick: () -> Unit
-) {
-    Row(
-        modifier = Modifier
-            .clickable(
-                onClickLabel = tabTitleText,
-                onClick = onClick
-            )
-            .padding(horizontal = 16.dp, vertical = 10.dp),
-        verticalAlignment = Alignment.CenterVertically,
-    ) {
-        Favicon(
-            url = url,
-            size = 24.dp,
-        )
-
-        Spacer(modifier = Modifier.width(32.dp))
-
-        Column {
-            PrimaryText(
-                text = tabTitleText,
-                modifier = Modifier.fillMaxWidth(),
-                fontSize = 16.sp,
-                maxLines = 1,
-            )
-
-            SecondaryText(
-                text = url,
-                modifier = Modifier.fillMaxWidth(),
-                fontSize = 12.sp,
```

Comment:
This is 14.sp in the `ListItem`. Can we double check with what is desired in the designs?

Reply:
I was in the middle of writing a comment 😅. You're too quick for me, @gabrielluong